### PR TITLE
Add template annotations to daemonset

### PR DESCRIPTION
**Pull request by kkonnov**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind feature

**What this PR does / why we need it**:

Ability to specify pod annotations in templates.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
NONE
**Does this PR introduce a user-facing change?**:
NONE

```release-note

```

### CODE REVIEW

**Comment by kkonnov** [comment]
Hello, @justinsb @nckturner Any news?

**Comment by kkonnov** [comment]
Hello guys. Any news?

**Comment by kkonnov** [comment]
Hello @hakman can you tell me do I need additional actions?

File: charts/aws-cloud-controller-manager/values.yaml [hunk 1]
```diff
@@ -112,6 +112,8 @@ securityContext: {}
 # podSecurityContext -- Pods Security Context.
 podSecurityContext: {}
 
+podAnnotations: {}
```
Comment (hakman) [inline charts/aws-cloud-controller-manager/values.yaml:115]:
```suggestion
# podAnnotations -- Pods Annotations.
podAnnotations: {}
```

**Review by hakman — APPROVED** [review]
Thanks @kkonnov!

**Pull request merged**